src/cros.rs

```diff
@@ -65,10 +66,22 @@ fi
 }
 
 pub fn setup_cros_repo(repo: &str, version: &str, reference: &Option<String>) -> Result<()> {
-    let url = if version == "tot" {
-        "https://chrome-internal.googlesource.com/chromeos/manifest-internal"
+    let config = Config::read()?;
+
+    let url = if config.is_internal() {
+        info!("Using internal repo urls");
+        if version == "tot" {
+            "https://chrome-internal.googlesource.com/chromeos/manifest-internal"
```

Comment:
probably adding a comment that references the official doc URL: https://chromium.googlesource.com/chromiumos/docs/+/master/developer_guide.md#Get-the-source-code to ensure that it is cited from the public source is a good idea ;)